samples/nrf9160/modem_shell/Kconfig

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Modem Shell config"
+
+config MOSH_IPERF3
+	bool "Enable iperf3 support"
+	default y
+	select NRF_IPERF3
+
+config MOSH_SOCK
+	bool "Socket tool support"
+	default y
+	help
+	  Socket tool with TCP, UDP, RAW, IPv4, IPv6 and packet socket support.
+
+config MOSH_PING
+	bool "ICMP ping support"
+	default y
+	help
+	  Ping is a tool for testing the reachability of a host on an IP network.
+
+config MOSH_LINK
+	bool "Link control support"
+	depends on LTE_LINK_CONTROL
+	depends on SETTINGS
+	default y
+	help
+	  Link control changes and queries the state of the LTE connection.
+
+config MOSH_CURL
+	bool "Enable CURL"
```

Comment:
Another enable:
```
	bool "cURL"
```

also cURL, not CURL.

Reply:
Fixed.